src/main/java/com/google/sps/servlets/KeywordServlet.java

```diff
@@ -0,0 +1,54 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Given a tweet or sentence, the servlet will extract salient terms from those
+ * and store them in a list.
+ */
+@WebServlet("/keyword")
+public class KeywordServlet extends HttpServlet {
+
+  private static final List<String> tweets = new ArrayList<>();
+  private static final List<String> terms = Arrays.asList("Black Lives Matter", "COVID-19");
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // TODO: Extract saliency
+    String json = mustacheJsonBuilder(terms);
+    response.getWriter().println(json);
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    final String tweet = request.getParameter("keyword-sentence");
+    if (tweet != null && !tweet.equals("")) {
+      tweets.add(request.getParameter("keyword-sentence"));
+    }
+    response.sendRedirect("/index.html");
+  }
+
+  /**
+   * Generates and returns a JSON usable by Mustache.
+   */
+  private String mustacheJsonBuilder(Collection<String> toJson) {
+    String builder = "{\"keywords\": [";
+    for (String term : toJson) {
+      builder += "{\"term\": \"" + term + "\"},";
+    }
+
+    // Remove extra comma at the end.
+    builder = builder.substring(0, builder.length() - 1) + "]}";
```

Comment:
Just to confirm, this is Java 7 right?  I was thinking about suggestion the join function but I think it only works on lists in Java 8
https://reversecoding.net/java-8-convert-list-string-comma/

Reply:
Yea, but I feel like using the join() function makes this less intuitive/readable to the reader, since it'd look like
`String.join("},{\"term\": ", toJson);` which I feel like is harder to read than simply String.format(//stuffs here).

Reply:
Assuming the join syntax works on lists you can just join on ",".  In Java 8 I would do this

```
List<String> terms = new ArrayList<string>();
for (String term : toJson) {
  terms.add(String.format("{\"term\": \"%s\"}", term));
}
String keywordPrefix = "{\"keywords\": [";
return String.format("{\"keywords\": [%s]}", terms.join(","))
```

You might also want to factor a function that, given a key and value return "{\"key\": value}" to make it easier to build JSON objects.  Alternatively, you can use a library that builds JSON for you.  A quick google search turns up 
this: https://docs.oracle.com/javaee/7/api/javax/json/JsonObjectBuilder.html
and this: https://github.com/stleary/JSON-java